src/xcode/ENA/ENA/Source/Services/__tests__/ExposureSubmissionServiceTests.swift

```diff
@@ -401,7 +401,8 @@ class ExposureSubmissionServiceTests: XCTestCase {
 		let client = ClientMock()
 		store.registrationToken = "dummyRegistrationToken"
 
-		let testResult = TestResult.allCases.randomElement() ?? TestResult.positive
+		let storedResults: [TestResult] = [.pending, .negative, .positive, .invalid]
+		let testResult = storedResults.randomElement() ?? TestResult.positive
```

Comment:
Wouldn't it be great to have a separate test for every test result case, so we can immediately see that something is wrong and don't have to wait that `.randomElement()` chooses a certain case? I think we've just experienced the drawback of this random approach.

Reply:
Added all cases as a test